profiles/multisite_drupal_communities/modules/custom/multisite_og_button/multisite_og_button.test

```diff
@@ -16,126 +24,133 @@ class OGButtonTestCase extends DrupalWebTestCase {
+    // Enable any modules required for the test.
     parent::setUp(array(
-    	'multisite_og_button', 
-    	'og'
+        'multisite_og_button',
+        'og',
     ));
-    
-    // create a group type
+
+    // Create a group type.
     $this->drupalCreateContentType(array(
-    	'type' => 'groups',
-      'name' => 'groups', 
+        'type' => 'groups',
+      'name' => 'groups',
     ));
     og_create_field(OG_GROUP_FIELD, 'node', 'groups');
-        
-    // Create some group content types
+
+    // Create some group content types.
     $this->drupalCreateContentType(array(
-    	'type' => 'group_content_a',
-      'name' => 'group_content_a', 
+        'type' => 'group_content_a',
+      'name' => 'group_content_a',
     ));
-    og_create_field(OG_AUDIENCE_FIELD, 'node', 'group_content_a');    
+    og_create_field(OG_AUDIENCE_FIELD, 'node', 'group_content_a');
 
     $this->drupalCreateContentType(array(
-    	'type' => 'group_content_b',
-      'name' => 'group_content_b', 
+        'type' => 'group_content_b',
+      'name' => 'group_content_b',
     ));
     og_create_field(OG_AUDIENCE_FIELD, 'node', 'group_content_b');
-    
+
     $this->drupalCreateContentType(array(
-    	'type' => 'group_content_c',
-      'name' => 'group_content_c', 
+        'type' => 'group_content_c',
+      'name' => 'group_content_c',
     ));
     og_create_field(OG_AUDIENCE_FIELD, 'node', 'group_content_c');
-    
+
     // Create a user who can edit groups.
     $this->users['admin'] = $this->drupalCreateUser(array(
       'access content',
       'administer blocks',
       'administer nodes',
       'administer users',
-			'administer content types',
+    'administer content types',
       'create groups content',
       'edit any groups content',
       'administer group',
     ));
-    $this->drupalLogin($this->users['admin']);    
-    
-    // Create some groups 
+    $this->drupalLogin($this->users['admin']);
+
+    // Create some groups.
     $settings = array();
     $settings['type'] = 'groups';
     $settings['title'] = 'Group A';
     $settings[OG_GROUP_FIELD][LANGUAGE_NONE][0]['value'] = 1;
     $this->groups['groupa'] = $this->drupalCreateNode($settings);
-    
+
     $settings = array();
     $settings['type'] = 'groups';
     $settings['title'] = 'Group B';
     $settings[OG_GROUP_FIELD][LANGUAGE_NONE][0]['value'] = 1;
     $this->groups['groupb'] = $this->drupalCreateNode($settings);
 
-    // Create a user who can edit group_content_a and group_content_b
+    // Create a user who can edit group_content_a and group_content_b.
     $this->users['usera'] = $this->drupalCreateUser(array(
       'access content',
       'create group_content_a content',
-			'create group_content_b content',                  
-      'edit any group_content_a content',                 
-      'edit any group_content_b content'
+    'create group_content_b content',
+      'edit any group_content_a content',
+      'edit any group_content_b content',
```

Comment:
This indentation is also not correct, it jumps back. This also happens a few times in this file.